packages/vite-config/src/vitest-config.ts

```diff
@@ -0,0 +1,22 @@
+import { defineConfig, mergeConfig } from "vitest/config";
+import viteConfig from "./vite-config.js";
+
+export default mergeConfig(
```

Comment:
Extend the shared Vite config to create a Vitest config that all of the apps can use